R/utils.R

```diff
@@ -1,17 +1,17 @@
-library("dplyr")
-
-# function to return folder name
+#' function to return folder name
+#' @param type Character string detailing type of widget to show
 get_filename = function(type) {
   filename = switch(type,
                     "Logistic growth rate" = "tree-logistic_growth_rate-2021-11-27.html", # nolint
                     "Simple logistic growth rate" = "tree-simple_logistic_growth_rate-2021-11-27.html", # nolint
                     "Simple trait log odds" = "tree-sim_trait_logodds-2021-11-27.html" # nolint
   )
-  filename = file.path("data", "wcdemo", "treeview", filename)
+  filename = file.path("www", "data", "treeview", filename)
```

Comment:
Haha, I thought this was a mistake and replaced it with system.file() ... only to break the app immediately. I'm guessing these are served files. This function is fine.